### ISSUE TITLE
Sliders don't fall off properly on fail

### ISSUE THREAD

**Issue by tpenguinltg** [itdelatrisu/opsu#208]
The slider ends fall off on fail, but not the ramps themselves:

![screenshot_2016-12-08_20-36-44](https://cloud.githubusercontent.com/assets/8697332/21034605/14f45ae2-bd87-11e6-8af6-cfa9c1cbc68d.png)

Possibly introduced by #205.

opsu commit 4b1dc39
Arch Linux x64, kernel 4.1-lts-ck

```
$ glxinfo  | grep -i opengl
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Intel(R) Ivybridge Mobile 
OpenGL core profile version string: 3.3 (Core Profile) Mesa 13.0.1
OpenGL core profile shading language version string: 3.30
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL core profile extensions:
OpenGL version string: 3.0 Mesa 13.0.1
OpenGL shading language version string: 1.30
OpenGL context flags: (none)
OpenGL extensions:
OpenGL ES profile version string: OpenGL ES 3.0 Mesa 13.0.1
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.00
OpenGL ES profile extensions:
```

**Comment by itdelatrisu** [comment]
Thanks for catching this.  It's happening on Windows too.  @mad-s, any ideas?

**Comment by mad-s** [comment]
Yes, this happens because the vertices of the slider are rendered as they are, i.e. not multiplied by Slick2D's view matrix.
I can fix it next Wednesday, if you want.

**Comment by itdelatrisu** [comment]
That'd be great, thanks so much! :)

**Comment by itdelatrisu** [comment]
Fixed in #212.
